rest-api/pom.xml

```diff
@@ -101,7 +101,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-json-org</artifactId>
-            <version>2.3.3</version>
+            <version>${jackson.version}</version>
```

Comment:
This only works if the exact dependency is listed in the dependency management section - I was aiming for a balance between moving some things to the dependency management section of the parent POM and not having to list every single dependency there.